go.mod

```diff
@@ -401,6 +452,8 @@ replace (
 	github.com/prometheus/client_model => github.com/prometheus/client_model v0.2.0
 	github.com/prometheus/common => github.com/prometheus/common v0.10.0
 	github.com/prometheus/procfs => github.com/prometheus/procfs v0.2.0
+	github.com/qri-io/starlib => github.com/qri-io/starlib v0.4.2-0.20200213133954-ff2e8cd5ef8d
```

Comment:
this is bringing in several unexpected dependencies (github.com/paulmach/orb, github.com/360EntSecGroup-Skylar/excelize, etc). is it required?

Reply:
Done (we forked a wee bit of code and got rid of the transdeps).